redesign/test.py

```diff
@@ -0,0 +1,6 @@
+import dataprofiler as dp
+import generator_builder as gb
+
+profile = dp.Profiler(dp.Data("./test.csv"))
+data = gb.Generator(profile).synthesize()
+print(data)
```

Comment:
- new line EOF
- we'll want to also use unittest and do some `self.assertEqual` to ensure the output is what we expect

Reply:
- also test error cases ... `with pytest.raise:` to ensure if we pass incorrect parameters that it raises the correct error